sandbox/src/extensions/ft.rs

```diff
@@ -0,0 +1,91 @@
+use near_sdk::{AccountIdRef, Gas, NearToken, json_types::U128, serde_json::json};
+
+use crate::{Account, SigningAccount, tx::FnCallBuilder};
+
+#[allow(async_fn_in_trait)]
+pub trait FtExt {
+    async fn ft_transfer(
+        &self,
+        token_id: &AccountIdRef,
+        receiver_id: &AccountIdRef,
+        amount: u128,
+        memo: Option<String>,
+    ) -> anyhow::Result<()>;
+
+    async fn ft_transfer_call(
+        &self,
+        token_id: &AccountIdRef,
+        receiver_id: &AccountIdRef,
+        amount: u128,
+        memo: Option<String>,
+        msg: &str,
+    ) -> anyhow::Result<u128>;
+}
```

Comment:
Can you also add here `async fn ft_on_transfer(sender_id, receiver_id, amount, msg)`?